.github/workflows/bach-tests.yml

```
@@ -90,11 +92,25 @@ jobs:
       - name: Install bigquery dependencies
         run: |
           pip install -e bach[bigquery]
-      - name: Unit tests big-query
+      - name: Unit tests BigQuery
```

or capitalize `postgres` in lines 80 and 84 :P just for following the same pattern
```suggestion
      - name: Unit tests bigquery
```

---

04c812b3496f9484445e9f6fca94a167ae0f15f3